src/System.Private.Xml.Linq/src/Resources/System.Private.Xml.Linq.rd.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
   <Library Name="System.Xml.Linq">
-    <Assembly Name="System.Private.Xml.Linq">
+    <Assembly Name="System.Xml.Linq">
```

Comment:
Why are you changing this - this is probably wrong. The assembly this is in si called System.Private.Xml.Linq (at least according to the project file).